data/tmpl/utility/content5.html

```diff
@@ -13,7 +13,8 @@
       <main>
         <h1>Free to Roam</h1>
         <p>Take your history, bookmarks and passwords with you.</p>
-        <button class="button">Get Firefox on your phone.</button>
+        <button class="button mobile" target="_blank" onclick="window.open('https://itunes.apple.com/us/app/apple-store/id989804926?pt=373246&mt=8&ct=firefox-onboarding-addon', '_blank')">Get Firefox for iOS.</button>
```

Comment:
I would grab the badges from here:
https://github.com/mozilla/bedrock/tree/master/media/img/l10n/en-US/firefox/android
and here
https://github.com/mozilla/bedrock/tree/master/media/img/l10n/en-US/firefox/ios

They are the official latest badges and have already been optimized.